src/Agent.Listener/CommandSettings.cs

```diff
@@ -127,6 +127,39 @@ public bool GetRunAsService()
                 defaultValue: false);
         }
 
+        public bool GetEnableAutoLogon()
+        {
+            return TestFlagOrPrompt(
+                name: Constants.Agent.CommandLine.Flags.EnableAutoLogon,
+                description: StringUtil.Loc("EnableAutoLogon"),
+                defaultValue: !Unattended);
+        }
+
+        public bool GetRestartPermission()
+        {
+            return TestFlagOrPrompt(
+                name: Constants.Agent.CommandLine.Flags.RestartIfNeeded,
+                description: StringUtil.Loc("RestartNow"),
+                defaultValue: true);
+        }
+
+        public bool GetContinueWithoutAutoLogon()
+        {
+            return TestFlagOrPrompt(
+                name: Constants.Agent.CommandLine.Flags.ContinueWithoutAutoLogon,
+                description: StringUtil.Loc("ContinueWithoutAutoLogon"),
+                defaultValue: false);
+        }
+
+        public string GetAutoLogonUserName()
+        {
+            return GetArgOrPrompt(
+                name: Constants.Agent.CommandLine.Args.WindowsLogonAccount,
+                description: StringUtil.Loc("AutoLogonAccountNameDescription"),
```

Comment:
where do you see the issue? The other names are also similar i.e., 'WindowsLogonAccountNameDescription'. Am I missing something?